9c-internal/chart/templates/data-provider-db.yaml

```diff
@@ -0,0 +1,147 @@
+        - name: NC_NoMiner
+          value: 'true'
+        - name: NC_BlockIndexFilePath
+          value: /data
+        - name: NC_Render
+          value: 'true'
+        - name: NC_BlockInsertInterval
+          value: '10'
+        - name: NC_TrustedAppProtocolVersionSigners__0
+          value: {{ $.Values.trustedAppProtocolVersionSigner }}
+        - name: NC_GenesisBlockPath
+          value: {{ $.Values.genesisBlockPath }}
+        - name: NC_MySqlConnectionString
+          valueFrom:
+            secretKeyRef:
+              key: write-value
+              name: data-provider-conn-aurora
+        - name: DP_HOST
+          valueFrom:
+            secretKeyRef:
+              key: write-host
+              name: data-provider-conn-aurora
+        - name: DP_USER
+          valueFrom:
+            secretKeyRef:
+              key: user
+              name: data-provider-conn-aurora
+        - name: DP_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: token
+              name: data-provider-conn-aurora
+        - name: DP_PORT
+          valueFrom:
+            secretKeyRef:
+              key: port
+              name: data-provider-conn-aurora
+        - name: DP_DATABASE
+          valueFrom:
+            secretKeyRef:
+              key: database
+              name: data-provider-conn-aurora
+        image: {{ $.Values.dataProvider.image.repository }}:{{ $.Values.dataProvider.image.tag }}
+        imagePullPolicy: {{ $.Values.dataProvider.image.pullPolicy }}
+        livenessProbe:
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - "set -e\nlast_block=\"$(\n  curl \\\n    -H 'Content-Type: application/json'\
+              \ \\\n    --data '{\"query\":\"query{chainQuery{blockQuery{blocks(desc:true,limit:1){timestamp}}}}\"\
+              }' \\\n    http://localhost:80/graphql_headless \\\n  | jq -r '.data.chainQuery.blockQuery.blocks[0].timestamp'\n\
+              )\"\nlast_timestamp=\"$(date +%s -u --date=\"$last_block\")\"\nnow=\"\
+              $(date +%s -u)\"\n[[ $(( now - last_timestamp )) -lt 500 ]]\n"
+          failureThreshold: 5
+          initialDelaySeconds: 1800
+          periodSeconds: 30
+          timeoutSeconds: 30
+        name: {{ $.Chart.Name }}-data-provider-db
+        ports:
+        - containerPort: 80
+          name: graphql
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 1300m
+            memory: 7Gi
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /bin/check_chain_tip.sh
+          name: {{ $.Chart.Name }}-data-provider-script
+          readOnly: true
+          subPath: check_chain_tip.sh
+        - mountPath: /data
+          name: {{ $.Chart.Name }}-data-provider-db-data
+      dnsPolicy: ClusterFirst
+      nodeSelector:
+        alpha.eksctl.io/nodegroup-name: 9c-main-m6g-2xl
```

Comment:
we are not using nodeSelector for now, so we have to remove or use valid one

Reply:
I delete it